clang/test/dpct/helper_files_ref/include/math.hpp

```diff
@@ -149,6 +149,19 @@ compare(const T a, const T b, const BinaryOperation binary_op) {
   return {compare(a[0], b[0], binary_op), compare(a[1], b[1], binary_op)};
 }
 
+/// Performs 2 elements comparison and returns an unsigned int.
```

Comment:
Performs 2 elements comparison, compare result of each element is  0 (false) or 0xffff (true), returns an unsigned int by composing compare result of two elements.

Reply:
fixed